jasmine/spec/inverted-index-test.js

```diff
@@ -1,74 +1,95 @@
 const InvertedIndex = require("../../src/js/inverted-index.js");
+
 describe('test functionality', () => {
-const path1 = [{
-      "text": "Alice falls into a rabbit hole and enters a world full of imagination.",
-      "title": "Alice in Wonderland"
-    },
-    {
-      "text": "An unusual alliance of man, elf, dwarf, wizard and hobbit seek to destroy a powerful ring.",
-      "title": "The Lord of the Rings: The Fellowship of the Ring."
-    }];
-  const path2 = __dirname +'/books.json';
-  const path3 =  __dirname +'/wysla.json';
-  const path4 =  __dirname +'/invalid.json';
-  const path5 =  __dirname +'/empty.json';
-  const path6 =  __dirname +'/notExist.json';
-  const path7 =  __dirname +'/incomplete.json';
+  const path1 = [{
```

Comment:
You should also abstract all these to a separate file like you've done for the other files i.e `path 1 - n`